src/nvim/testdir/setup.vim

```diff
@@ -23,25 +70,6 @@ set wildoptions=
 " Prevent Nvim log from writing to stderr.
 let $NVIM_LOG_FILE = exists($NVIM_LOG_FILE) ? $NVIM_LOG_FILE : 'Xnvim.log'
 
-
-" Make sure 'runtimepath' and 'packpath' does not include $HOME.
-set rtp=$VIM/vimfiles,$VIMRUNTIME,$VIM/vimfiles/after
-let &packpath = &rtp
-
-" Avoid storing shell history.
-let $HISTFILE = ""
-
-" Make sure $HOME does not get read or written.
-let $HOME = expand(getcwd() . '/XfakeHOME')
-if !isdirectory($HOME)
-  call mkdir($HOME)
-endif
-
-" Use default shell on Windows to avoid segfault, caused by TUI
-if has('win32')
-  let $SHELL = ''
-  let $TERM = ''
-  let &shell = empty($COMSPEC) ? exepath('cmd.exe') : $COMSPEC
-  set shellcmdflag=/s/c shellxquote=\" shellredir=>%s\ 2>&1
-  let &shellpipe = &shellredir
-endif
+" Don't depend on system locale, always use utf-8.
+" Ref: https://github.com/neovim/neovim/pull/2929
+set encoding=utf-8
```

Comment:
`set encoding` does nothing in Nvim. The reference to https://github.com/neovim/neovim/pull/2929 is outdated, that PR was before we made 'encoding' readonly.

So this line isn't needed unless it's in Vim.